src/tests/src/solver/expressions/test_expressions.cpp

```diff
@@ -406,16 +406,17 @@ class SubstitutionVisitor: public CloneVisitor
                                 component_variable_node);
             it != ctx_.variables.end())
         {
-            return const_cast<Node*>(dynamic_cast<const Node*>(&(*it)));
+            return &*it;
         }
+
         else
         {
             return CloneVisitor::visit(component_variable_node);
         }
     }
 };
 
-void fillContext(ANTLRContext& ctx)
+void fillContext(ANTLRContext& ctx, Registry<Node>& registry)
```

Comment:
It doesn't seem to be necessary to pass a registry here

Reply:
yes, from an old commit